python/api/application.py

```diff
@@ -56,9 +57,15 @@ def default(self, obj):
 #Allow cross-origin requests. TODO should eventually lock down the permissions on this a bit more strictly, though only allowing GET requests is a good start.
 CORS(application, resources={r"/api/*": {"origins": "*"}}, methods=['GET'])
 
+#Allow us to test locally if desired
+if 'docker' in sys.argv:
+    database_choice = 'docker_database'
+else:
+    database_choice = 'remote_database'
+
 with open('secrets.json') as f:
```

Comment:
@jkwening Good point. You should actually just copy-paste your secrets.json over into the api folder (duplicated). It doesn't show up in git because secrets is excluded. 

Double maintenance of secrets will be resolved when we refactor this in the future - eventually the API server will need access to the whole python folder so that it can run data collection jobs.